### DISCUSSION TITLE
snw-release/

### DISCUSSION THREAD

**Discussion by giscus** [narwhrl/narwhrl.github.io#8] [Giscus]
# snw-release/

h3{         font-family: 'Noto Sans Serif', serif !important     }      .center {         margin: auto;         width: 50%;         padding: 10px;     }                               SEASON 1

https://startrekcn.cn/snw-release/

**Comment by Bashir-1** [comment]
**夸克盘的链接放错了，点进去变神童了**

**Reply by narwhrl** [reply]
已修复

**Comment by fan881217** [comment]
首页看不到某个具体的集之后就找不到这集的1080P下载了

**Reply by amos1236** [reply]
+1，新世界S02，遇到同样的问题
